src/libraries/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -45,8 +45,7 @@ public void RelativeSearchPath_Is_Null()
 
         [Fact]
         [SkipOnPlatform(TestPlatforms.Browser, "throws pNSE")]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/49868", TestPlatforms.Android)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/36896", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
+        [SkipOnPlatform(TestPlatforms.Android | TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst, "The dotnet process will not installed on device")]
```

Comment:
Nit: Something's off in the grammar of the justification.

Reply:
we can also combine this with the `TestPlatforms.Browser` because it's the same underlying reason